assignment-1-sql/chimi/chimi-sherpa.sql

```diff
@@ -0,0 +1,183 @@
+-- Week 2 Assignment --
+-- QUESTION 1 ---------------------------------------------
+
+SELECT *
+FROM fact_order
+WHERE charge_amount > 500
+AND completed_at is NOT null
+
+ORDER BY completed_at desc
+LIMIT 1
+
+
+-- Question 2 --------------------------------------------
+
+SELECT 
+    DATE_TRUNC('week', fo.completed_at) as period,
+
+COUNT(DISTINCT fo.id) AS order_volume,
+SUM(fo.total_charged_amt) AS GMV
+
+FROM fact_order fo 
+JOIN zones z ON z.id = fo.zone_id
```

Comment:
fact_order actually already has a country_id, so no need to do the joins here.

Reply:
I know the assignment talks about joins and these other tables but if we can avoid it it's always more efficient :)